src/site.rs

```diff
@@ -346,7 +340,6 @@ impl Site {
 
     /// Builds the site to the `public` directory after deleting it
     pub fn build(&self) -> Result<()> {
-        self.clean()?;
```

Comment:
We want to be able to do a clean build too though. Maybe add an argument to `build` called `clean_first`?